deploy/scripts/src/ais/requestGenerator/aisReqFormat.ts

```diff
@@ -0,0 +1,19 @@
+export interface TICF_ACCOUNT_INTERVENTION {
```

Comment:
```suggestion
export interface TicfAccountIntervention {
```
Prefer PascalCase for types/interfaces

Reply:
Done